src/components/primitives/HtmlText/HtmlText.tsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+
+interface Props {
+    html: any;
```

Comment:
I think html will always be a string here

Reply:
Changed to string